test/t8_IO/t8_gtest_vtk_writer.cxx

```diff
@@ -75,6 +77,42 @@ destroy_grid<t8_forest_t> (t8_forest_t *forest)
   t8_forest_unref (forest);
 }
 
+template <typename grid_t>
+int
```

Comment:
Does this work?
```suggestion
static int
```

Reply:
It works for the template, but not for explicit template specializations